lib/pixify.js

```diff
@@ -118,8 +128,8 @@ module.exports = function(options, callback) {
                 }))
                 .pipe(gulpif(options.compress, uglify()))
                 .pipe(header(license, {
-                    date: (new Date()).toString(),
-                    name: packageInfo.name,
+                    date: (new Date()).toUTCString(),
+                    name: options.name,
```

Comment:
Modified this, too, because the package name is in pkg already and this allows to reuse the name option properly.